src/voku/helper/AntiXSS.php

```diff
@@ -2014,9 +2014,13 @@ public function setStripe4byteChars($bool): self
      *    vulnerabilities along with a few other hacks I've
      *    harvested from examining vulnerabilities in other programs.
      *
-     * @param array|mixed $str <p>input data e.g. string or array of strings</p>
+     * @param string[]|string $str <p>input data e.g. string or array of strings</p>
      *
-     * @return mixed
+     * @return string
+     *
+     * @psalm-template T of string[]|string
```

Comment:
do you prefer to use these prefixed annotations, or should I replace the main `@param` and `@return` docs?